docs/topics/crud-experience-intro.adoc

```diff
@@ -0,0 +1,24 @@
+The {ProductShortName} CRUD application Booster expands on the HTTP API Booster to provide a basic example of performing _create_, _read_, _update_ and _delete_ (_CRUD_) operations on a PostgreSQL database using a simple HTTP API.
+
+The Booster also demonstrates the ability of the HTTP application to locate and connect to a database. Each runtime determines in an opinionated manner to implement the connectivity solution that is best suited in the given case. The runtime can choose between using _JDBC_, _JPA_, or _ORM_.
```

Comment:
@cescoffier does this information still apply, or are all the quickstarts using JDB for database connectivity at the moment?